ads/navegg.md

```diff
@@ -30,6 +30,7 @@ If you want to serve ads with Navegg data you need to use this adapter.
 </amp-ad>
 
 ```
+note: It is common to see a no-content ad in this example.
```

Comment:
oops. Sorry I meant to add this line to the ads.amp.html example page. So when engineers test the ads page, they know it's common to see no fill ad.  (That's also why I encourage to put an always rendered fake ad there, so that it helps up to detect issues once we see no ad fill).